content/reference/user/organization/index.md

```diff
@@ -77,3 +79,19 @@ Plans view history.
 * **From** - Start date of the plan.
 * **To** - End date of the plan, if there is one.
 * **Credits** - Number of credits awarded each month by the plan.
+
+## Offers
+
+{{< alert warning >}}
+This section is only available to System Admins.
```

Comment:
```suggestion
This section is only available to global System Admins.
```

Reply:
Maybe explain what we mean by "global System Admins" (= System Admin on Team *)?